centreon/www/include/configuration/configObject/contact/DB-Func.php

```diff
@@ -197,6 +197,49 @@ function disableContactInDB($contact_id = null, $contact_arr = array())
     }
 }
 
+/**
+ * Unblock contacts in the database
+ *
+ * @param int|array<int, string>|null $contact Contact ID, array of contact IDs or null to unblock all contacts
+ */
+function unblockContactInDB(int|array|null $contact = null): void
+{
+    global $pearDB, $centreon;
+
+    if (null === $contact || [] === $contact) {
+        return;
+    }
+
+    if (is_int($contact)) {
```

Comment:
Not consistent with the develop PR https://github.com/centreon/centreon/pull/1299

Reply:
My mistake : I didn't check the right code. 😭
Sorry 🤗